pandas/tests/indexes/multi/test_constructors.py

```diff
@@ -647,6 +648,22 @@ def test_from_frame():
     result = MultiIndex.from_frame(df)
     tm.assert_index_equal(expected, result)
 
+def test_from_frame_multiIndex():
```

Comment:
You'll need to a `pytest.mark.skipif(pa_version_under1p01)` decorator here to skip if pyarrow is not installed